### DISCUSSION TITLE
Ruby gem checksum mismatch on GitHub Packages

### DISCUSSION THREAD

**Discussion by exAspArk** [community/community#47659] [Packages] [answered]
### Select Topic Area

Bug

### Body

We suddenly started receiving errors when trying to install previously published private Ruby gem versions:

```
Retrying fetcher due to error (4/4): Bundler::HTTPError The checksum of /info/GEM_NAME does not match the checksum provided by the server! Something is wrong (local checksum is "\"f833ea5c2ad9f578a6c94f3e647e9e9f\"", was expecting "\"47fcd689bf3811a0fc093b9f2691d99d\"").
```

Yesterday, we were able to successfully install the same dependency versions.

The last time we had a similar issue, it was related to https://github.com/orgs/community/discussions/43861 (`Malformed version number string`)

**Comment by exAspArk** [comment, marked as answer]
It looks like it was resolved around 11am EST

> Our engineers believe it should be resolved now.
> – GH support

**Comment by taltcher** [comment]
@exAspArk  We still experiencing this issue - is it fully resolved?

**Reply by taltcher** [reply]
Do we need to re-create a new version for a package or something?

**Comment by Akash1134** [comment]
GitHub Actions and Packages are now separate categories. While both of their features play well together, they are separate products and we want to give each of them their own space for questions and feedback. Keeping that in mind, this discussion has been moved over from the [Actions](https://github.com/orgs/community/discussions/categories/actions) category.
